beaconChain/src/helpers/stateTransitionHelpers.ts

```diff
@@ -119,7 +133,7 @@ export function getActiveValidatorIndices(validators: Validator[], epoch: Epoch)
 function shuffle<T>(values: T[], seed: bytes32): T[] {
   const valuesCount: int = values.length;
   // Entropy is consumed from the seed in 3-byte (24 bit) chunks.
-  const randBytes: number = 3;
+  const randBytes = 3;
```

Comment:
why did you remove `: number`

Reply:
This was removed by a eslint --fix.
I think theres a rule to the effect of: no super obvious types.
something like `const b: boolean = true;` would likewise get flagged. (see the isPowerOfTwo function where that happened and was changed)

Reply:
That actually makes sense...